### DISCUSSION TITLE
possibility to run remix app as subdir of another web?

### DISCUSSION THREAD

**Discussion by palmic** [remix-run/remix#2448] [Q&A] [answered]
Hello,
i am newbie and i am now investigating if there is possibility to run remix app as subdir of another web
example: **www.hostname/mycoolremixapp**

So everything would behave as root of the app under that /mycoolremixapp url, however public assets in generated html would be still linked properly.

(i am asking just about how to set remix app, not outer webserver etc..)

Thank you for any reply.

**Comment by kiliman** [comment, marked as answer]
Check out https://github.com/kiliman/remix-mount-routes

**Reply by palmic** [reply]
Thank you!

BTW. you have typo in README:

const { mountRoutes } = require('remix-mountroutes')
should be
const { mountRoutes } = require('remix-mount-routes')
